added another choose example

Nihvel opened the pull request:
more choose can be used together for creating extended conditions which do not depend from each others

## Proposed change
I added an example that shows how more `choose` can be used within the same automation or script.



## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Nihvel committed:
added another choose example

more choose can be used together for creating extended conditions which do not depend from each others

homeassistant commented:
Hi @Nihvel, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16972).

Once you do that we will be able to review and accept this pull request.

Thanks!

frenck commented on source/_docs/scripts.markdown:
````diff
@@ -610,6 +610,63 @@ automation:
+{% raw %}
+
+```yaml
+automation:
````
Please adjust the example to comply with our YAML style guide.

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Nihvel replied:
Done it, must be correct now

frenck commented:
I think the example needs a bit more explanation or comments. Due to its size, it is hard to follow for someone new to the syntax.

Nihvel committed:
example fixed with proper yaml syntax

Nihvel committed:
more comments added

Extended explanation was added to the example to help understand the automation. The automation was cleaned from unnecessary conditions

Nihvel committed:
typo

pipeline failed due to possible typo

frenck commented on source/_docs/scripts.markdown:
```diff
@@ -610,6 +610,62 @@ automation:
 {% endraw %}
 
+
+More `choose` can be used together. This is the case of an IF-IF. 
+The following example shows how a single automation can control entities that aren't related to each other but have in common the same trigger.
+When the sun goes below the horizon, the `porch` and `garden` lights must turn on. If someone is watching the TV in the living room, there is a high chance that someone is in that room, therefore the living room lights have to turn on too. Same concept applies for the `studio` room.
```
```suggestion
More `choose` can be used together. This is the case of an IF-IF.

The following example shows how a single automation can control entities that aren't related to each other but have in common the same trigger.

When the sun goes below the horizon, the `porch` and `garden` lights must turn on. If someone is watching the TV in the living room, there is a high chance that someone is in that room, therefore the living room lights have to turn on too. The same concept applies to the `studio` room.
```

frenck commented on source/_docs/scripts.markdown:
```diff
@@ -610,6 +610,62 @@ automation:
+            - conditions:
+                - condition: state
+                  entity_id: binary_sensor.studio_pc
+                  state: on
```
```suggestion
                  state: "on"
```

frenck commented on source/_docs/scripts.markdown:
```diff
@@ -610,6 +610,62 @@ automation:
+            - conditions:
+                - condition: state
+                  entity_id: binary_sensor.livingroom_tv
+                  state: on
```
```suggestion
                  state: "on"
```

frenck commented on source/_docs/scripts.markdown:
```diff
@@ -610,6 +610,62 @@ automation:
+automation:
+  - alias: "Turn lights on when the sun gets dim and if some room is occupied"
+      trigger:
+        - platform: numeric_state
+          entity_id: sun.sun
+          value_template: "{{ state.attributes.elevation }}"
+          below: 4
```
```suggestion
        - platform: numeric_state
          entity_id: sun.sun
          attribute: elevation
          below: 4
```

Nihvel committed:
Update source/_docs/scripts.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Nihvel committed:
Update source/_docs/scripts.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Nihvel committed:
Update source/_docs/scripts.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Nihvel committed:
Update source/_docs/scripts.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck reviewed (approved):
Thanks, @Nihvel 👍
